### DISCUSSION TITLE
Uncaught error while Cypress types in on google autocomplete input

### DISCUSSION THREAD

**Discussion by piotr-bejamas** [cypress-io/cypress#14935] [Component Testing]
While interacting with google autocomplete search sometimes I receive uncaught exception when typing (I can see that phrase prints in input, sec before error) and page becomes blank. Rarely it passes, but couple commands later Chrome crashes with "Oh snap", code error 11.

App: react + gatsby
Pieces of codes, I tried:
```
cy.getBySel('booking-location-title').should('contain', data.de.step1Title);
cy.getBySel('booking-location-search-input').type(data.de.step1searchQuery)
cy.get('.pac-container', {timeout: 10000}).should('exist').and('be.visible');
cy.get('.pac-item').first().should('be.visible').click()
cy.getBySel('booking-location-distance').should('exist').and('be.visible')
```
or
```
cy.getBySel('booking-location-title').should('contain', data.de.step1Title);
cy.getBySel('booking-location-search-input').type(`${data.de.step1searchQuery}{downarrow}{enter}`)
cy.getBySel('booking-location-distance').should('exist').and('be.visible')
cy.getBySel('booking-location-name').first().should('contain', data.de.step1searchFirstResultTitle);
```
- Tried to ignore uncaught error by listening Cypress.on in support/index.js
- using keys instead clicking
- adding delay to type
- using gatsby-cypress and waitForRouteChange() in this spec

Error: Cannot destructure property 'targetLocation" of (intermediate value) as it is undefined.

**Comment by piotr-bejamas** [comment]
I narrowed issue to chromium based browsers. Works as expected on Firefox. Any ideas for chromium workaround?

**Comment by bahmutov** [comment]
Please create a reproducible example with full code and spec file, otherwise we cannot know what is going on with your code.
